Albo1125.Common/CommonLibrary/FriendlyKeys.cs

```diff
@@ -0,0 +1,221 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Albo1125.Common.CommonLibrary
+{
+    public static class FriendlyKeys
+    {
+        public static string GetFriendlyNames(Keys modifier, Keys key, bool format = true, char format_color = 'b')
+        {
+            string output = "";
+            if (modifier != null && modifier != Keys.None)
+                output += GetFriendlyName(modifier, format, format_color) + " + ";
```

Comment:
Always always ALWAYS use curly braces for code blocks, you'll make your code prone to bugs and unexpected behaviour which is hard to diagnose